package.json

```diff
@@ -15,13 +15,12 @@
         "@bazel/ibazel": "^0.10.3",
         "@commitlint/cli": "^8.0.0",
         "@commitlint/config-conventional": "^8.0.0",
+        "date-fns": "1.30.1",
         "@types/hammerjs": "2.0.35",
         "@types/jasmine": "~3.3.13",
         "@types/node": "7.0.18",
         "@types/webpack": "~4.4.35",
-        "@zeit/ncc": "^0.20.4",
```

Comment:
where did this go??

Reply:
I put all of the vendoring-only deps in /third_party/package.json. Keeps the main deps smaller and also ensures that we're not accidentally using pacakages from @npm that should be vendored such as `@npm//babel/core` (ran into that issue when working on this)